website/src/controllers/api-v2/groups.js

```diff
@@ -691,8 +691,9 @@ var inviteByEmails = function(invites, group, req, res, next){
 
 api.invite = function(req, res, next){
   var group = res.locals.group;
-
-  if (req.body.uuids) {
+  if (group.memberCount + group.invites.length >= 30) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
Could you please put `30` into a variable or similar that can be used in all places it's needed throughout the code base? There's an idea for how to do that here: https://github.com/HabitRPG/habitrpg/pull/6762#issuecomment-188060614
but you might have your own ideas. Feel free to ask if you'd like feedback or clarifications.